src/index.js

```diff
@@ -1,21 +1,43 @@
 import './style.css';
-import getData from './moduled/getData.js';
-
+import { getData, getLikes } from './modules/getData.js';
+import { addLike } from './modules/mealsLikes.js';
 // Get All meals Data
 getData()
   .then((data) => {
     let mealCard = '';
     data.meals.forEach((elem) => {
       mealCard += `  
-    <div class="card">
+      <div id="${elem.idMeal}" class="card">
     <img src="${elem.strMealThumb}" alt="Sample photo">
     <div class="text">
-      <h3>${elem.strMeal} ❤</h3>
-      <h5 class="likes">2 likes</h5>
-      <button>Comment</button>
+      <h3>${elem.strMeal} </h3>
+      <h4 class="likes-number">No likes</h4>
+      <button class="like-heart">Like 👍</button>
+      <button>Comment 💬</button>
     </div>
     </div>
 `;
+      // display likes number in homepage
+      getLikes().then((data) => {
+        const likes = document.querySelectorAll('.likes-number');
+        /* eslint-disable */
+        data.filter((item) => { 
+          likes.forEach((like) => { 
+            const cardId = like.parentElement.parentElement.id;
+            if (item.item_id === cardId) {
+              like.innerHTML = `${item.likes} likes`;
+            }
+          });
+        });
+        // get likes ID when like button clicked
+        const likeBtn = document.querySelectorAll('.like-heart');
+        for (let i = 0; i < likeBtn.length; i += 1) {
```

Comment:
Great job using a `for` loop to loop over the node list, but a more efficient and effective way is using the `forEach` method for less readable code. Kindly check this [https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach](url) for further deatails